docs/source/workflows/performance_workflows.rst

```diff
@@ -0,0 +1,86 @@
+Performance workflows
+=====================
+
+A performance workflow performs analysis on a module.
+Currently, the only analysis implemented is "cross validation analysis," which performs cross-validation on a predictor.
+
+The following example demonstrates how to use the python SDK to register a performance workflow, wait for validation to complete and check the final status:
+
+.. code:: python
+
+   from time import sleep
+   from citrine.informatics.workflows import PerformanceWorkflow
+   from citrine.informatics.analysis_configuration import CrossValidationAnalysisConfiguration
+
+   workflow = project.workflows.register(
+       PerformanceWorkflow(
+           name='Demo Performance Workflow',
+           analysis=CrossValidationAnalysisConfiguration(
+               name='analysis_settings',
+               n_folds=2,
+               n_trials=3,
+               max_rows=200,
+               seed=10,
+               group_by_keys=['~~x']
+           )
+       )
+   )
+
+   # wait until the workflow is no longer validating
+   # we must get the workflow every time we wish to fetch the latest status
+   while project.workflows.get(workflow.uid).status == "VALIDATING":
+       sleep(10)
+
+   # print final validation status
+   validated_workflow = project.workflows.get(workflow.uid)
+   print(validated_workflow.status)
+   # status info will contain relevant validation information
+   # (i.e. why the workflow is valid/invalid)
+   print(validated_workflow.status_info)
+
+Execution and results
+---------------------
+
+When a performance workflow is executed against a module, it performs the analysis and returns an "analysis results" dictionary that contains performance metrics.
```

Comment:
Note: I'm not sure if "analysis results dictionary" is the general way we want to present this to users, it's just the term that makes the most sense to me